src/shared/lib/FormatUtils.ts

```diff
@@ -45,3 +45,15 @@ export function toFixedWithThreshold(value: number, digits: number): string
 export function getPercentage(proportion: number, digits: number = 1) {
     return `${toFixedWithThreshold(100 * proportion, digits)}%`;
 }
+
+/* difference between this function and the previous one is it will display
+percentages less than 1% as <1%
+*/
+export function getMutSigPercentage(proportion: number, digits: number = 0) {//0.003 -> 0.3
```

Comment:
MutSig is actually a separate thing, I think its an algorithm or a suite of algorithms... for clarity write out 'Signature'